### PR TITLE
fix(controlplane): validate pipeline and chain weights

### LINKED ISSUE #2388: fix(controlplane): value ranges are not validated by the services
## Motivation

Several request fields are accepted by the services with values that the dataplane or the wire format cannot represent, and the result is silently wrong instead of rejected:

- VLAN id: any `uint32` is cast to `uint16` and written into the TCI unmasked, so `5000` sets PCP/DEI bits on egress — https://github.com/yanet-platform/yanet2/blob/620c131607d1feaf198e6135f8ca7845b8f4ca59/devices/vlan/controlplane/service.go#L49, https://github.com/yanet-platform/yanet2/blob/620c131607d1feaf198e6135f8ca7845b8f4ca59/devices/vlan/dataplane/dataplane.c#L111-L112.
- Device name: silently truncated to 80 bytes (`CP_DEVICE_NAME_LEN`) — https://github.com/yanet-platform/yanet2/blob/620c131607d1feaf198e6135f8ca7845b8f4ca59/lib/controlplane/config/cp_device.c#L23.
- nat64 prefix: only the byte count is checked, the request cannot express a length at all — https://github.com/yanet-platform/yanet2/blob/620c131607d1feaf198e6135f8ca7845b8f4ca59/modules/nat64/controlplane/service.go#L196-L217.
- MPLS label: `uint32` with no 20-bit bound — `modules/route-mpls/controlplane/routemplspb/v1/routempls.proto`.
- Ports in acl/fwstate sync config: `uint32` with no `0..=65535` check.
- Pipeline weight: `// TODO weight clamp` — https://github.com/yanet-platform/yanet2/blob/620c131607d1feaf198e6135f8ca7845b8f4ca59/lib/controlplane/config/zone.c#L727.
- pdump `snaplen 0`: silently replaced by the server default while the CLI reports success — https://github.com/yanet-platform/yanet2/blob/620c131607d1feaf198e6135f8ca7845b8f4ca59/modules/pdump/controlplane/service.go#L471-L474.

Expected: out-of-range values are rejected with `InvalidArgument` naming the field and the allowed range; the CLI mirrors the same bounds at parse time (tracked separately).

## Evidence

Code-proven as linked. The VLAN case is also observable end to end: `yanet-cli device vlan update --vlan 5000` is accepted by the client on `620c1316`.

## Scope

The services listed above. Client-side mirrors are in the CLI epic.

## Acceptance

- [ ] Each listed field rejects out-of-range values with `InvalidArgument` and a message naming the range.
- [ ] A Go test per service covers the boundary values.
- [ ] Proto comments state the accepted range for each field.

### PR DESCRIPTION
**Pull request by 3Hren**
- Reject function chain and device pipeline weight sums above 65535 with field-specific `InvalidArgument` errors before services access shared memory.
- Validate protobuf messages without reading or changing live configuration, checking device input and output independently and preserving zero weights.
- Document the accepted ranges and cover the boundaries in shared model tests with focused service checks.

Closes #2388.

### CODE REVIEW

**Comment by 3Hren** [comment]
@codex review

File: lib/controlplane/config/zone.c [hunk 1]
```diff
@@ -770,7 +806,24 @@ cp_config_update_devices(
 	struct cp_device *devices[],
 	yanet_error **err
 ) {
-	// TODO weight clamp
+	for (uint64_t idx = 0; idx < device_count; ++idx) {
+		struct cp_device *device = devices[idx];
+		if (cp_device_entry_validate_weights(
+			    ADDR_OF(&device->input_pipelines), err
```
[the remark this answers is not in the document]
Comment (3Hren) [reply lib/controlplane/config/zone.c]: Removed the C-level weight validation in 9d77e630. This PR now validates only protobuf requests before service FFI calls, so it no longer reads device lifetime state or changes configuration locking.

**Comment by 3Hren** [comment]
@codex review
